tests/cosmogony_lux_test.rs

```diff
@@ -87,3 +87,26 @@ fn test_lux_admin_levels_with_serialisation() {
 
     test_wrapper_for_lux_admin_levels(cosmogony_from_json);
 }
+
+#[test]
+fn test_lux_zone_types() {
+    // Check the zone types in the built cosmogony
+    let cosmogony = create_cosmogony_for_lux();
+    let zone_type_counts = cosmogony.meta.stats.zone_type_counts;
+
+    fn assert_count(counts: &BTreeMap<String, u64>, key: String, value: u64) {
```

Comment:
A very small detail, but we could directly define `key` as a `&str` here, and remove all the `format!` below.

Reply:
:+1: